index.js

```diff
@@ -57,6 +57,63 @@ bot.registerCommand(
     }
 );
 
+bot.registerCommand(
+    "createPoll",
+    (msg, args) => {
+
+        const emojiSet = args[0];
+        let pollTitle = [];
+        let optionSet = [];
+        let optionBuilder = [];
+        let argCount = 0;
```

Comment:
Defining all the variables at the top of the function is considered good style when you are using `var` declarations, since that's how the engine defines it anyways. However, with `const` and `let`, the place you define them is the place they actually get defined (there is no hoisting of const/let).

Because of that, I think it would be clearer if some of these were defined closer to where they are used, like moving `optionBuilder` down to where it is actually constructed.